### PR TITLE
Sandboxes

### PR DESCRIPTION
**Pull request by jezhumble**
I've added a new page for sandboxes, and revised the Infrastructure and AWS pages (the latter substantially) to reflect how I think AWS (both prod and sandboxes) should work in future. Please review!

### CODE REVIEW

File: _pages/infrastructure/aws.md [hunk 1]
```diff
@@ -3,53 +3,52 @@ title: Amazon Web Services
...
+### OS baseline
 
-Every inter-agency agreement (IAA) at 18F needs to have a line item on a total value to spend on infrastructure, including Amazon Web Services (AWS). Unless it is part of a negotiation with 18F Infrastructure, we *do not* pay for non-production hosting costs. All costs must go back to the Federal partner or another funding source. There is no actual concept of _non-billable_ - there are only costs that are directly or indirectly billable. If we don't bill a funding source, it means that 18F's rates must go up that next fiscal year in order indirectly recoup costs.
+We use a pre-hardened version of [Ubuntu](https://en.wikipedia.org/wiki/Ubuntu_%28operating_system%29) as our baseline OS for all EC2 instances in AWS. In AWS, there are [Amazon Machine Images](https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/AMIs.html) (AMIs) in each AWS Region that these controls already implemented. You should always launch new instances from this baseline.
```
Comment (afeld) [inline _pages/infrastructure/aws.md]: Can we add information (or a link) about how to find it?

Comment (afeld) [inline _pages/infrastructure/aws.md]: Also, it seems that this paragraph now exists on two pages.

**Comment by jezhumble** [comment]
@afeld - thanks very much for your feedback. I've updated the text: please do let me know if it addresses your concerns or if there's other stuff that needs more clarity.

**Comment by NoahKunin** [comment]
Looks great - merging!

**Comment by afeld** [comment]
Thanks for this, gents!

**Pull request merged**